<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>midifieravis</class>
 <widget class="QWidget" name="midifieravis">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>281</width>
    <height>374</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>271</y>
     <width>100</width>
     <height>41</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>modifier</string>
   </property>
  </widget>
  <widget class="QLabel" name="ajouteravis_2">
   <property name="geometry">
    <rect>
     <x>45</x>
     <y>10</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Modifier Client</string>
   </property>
  </widget>
  <widget class="QPushButton" name="retour">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>270</y>
     <width>100</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>retour</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_idclient_avis">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>86</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>CIN</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="id_client_avis">
   <property name="geometry">
    <rect>
     <x>107</x>
     <y>87</y>
     <width>108</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_id_avis">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>128</y>
     <width>66</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Nom</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="id_edit_avis">
   <property name="geometry">
    <rect>
     <x>107</x>
     <y>129</y>
     <width>108</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_avis">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>170</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Prenom</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_nom_film">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>212</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>E-mail</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="avis_edit">
   <property name="geometry">
    <rect>
     <x>107</x>
     <y>171</y>
     <width>108</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="film_edit">
   <property name="geometry">
    <rect>
     <x>107</x>
     <y>213</y>
     <width>108</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>330</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../Downloads/res/lagoralogo.png</pixmap>
   </property>
  </widget>
  <zorder>film_edit</zorder>
  <zorder>label_nom_film</zorder>
  <zorder>pushButton</zorder>
  <zorder>id_client_avis</zorder>
  <zorder>label_id_avis</zorder>
  <zorder>retour</zorder>
  <zorder>id_edit_avis</zorder>
  <zorder>avis_edit</zorder>
  <zorder>ajouteravis_2</zorder>
  <zorder>label_avis</zorder>
  <zorder>label_idclient_avis</zorder>
  <zorder>label</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
